lib/chef/provisioning/convergence_strategy/install_msi.rb

```diff
@@ -9,7 +9,7 @@ class InstallMsi < PrecreateChefObjects
 
       def initialize(convergence_options, config)
         super
-        @install_msi_url = convergence_options[:install_msi_url] || 'http://www.opscode.com/chef/install.msi'
+        @install_msi_url = convergence_options[:install_msi_url] || 'https://www.opscode.com/chef/install.msi'
```

Comment:
Note to either you or us: these URLs should now be https://chef.io/chef/install.msi and https://chef.io/chef/install.sh.  Otherwise, this looks great!